packages/types/src/clerk.ts

```diff
@@ -411,7 +411,7 @@ export interface Clerk {
    * {@link Clerk.client.signIn.authenticateWithRedirect} or {@link Clerk.client.signUp.authenticateWithRedirect}
    */
   handleRedirectCallback: (
-    params: HandleOAuthCallbackParams | HandleSamlCallbackParams,
+    params: HandleOAuthCallbackParams,
```

Comment:
❓ Why did we remove the `HandleSamlCallbackParams` ? I think it is intentional to keep both of them.

Reply:
This was automatically removed by eslint because the two types are the same - I will revert the change for now